storage/directory.go

```diff
@@ -102,7 +102,7 @@ type PutDirectoryInput struct {
 	DirectoryName string
 }
 
-// Put puts a directoy into the Triton Object Storage service is an idempotent
+// Put puts a director into the Triton Object Storage service is an idempotent
```

Comment:
This was still spelled wrong, forgot the `y`. 😄

Reply:
Oh! That's my bad, sincere apologies!

I will get it fixed alongside different Pull Request.